installer/nginx/libretime.conf

```diff
@@ -12,6 +12,10 @@ server {
   client_max_body_size 512M;
   client_body_timeout 300s;
 
+  location /hls {
+     alias /var/lib/libretime/playout/hls;
```

Comment:
So this assumes that there is a common volume between the nginx server and liquidsoap. This is not currently required and requires documenting. If HLS is going to be a default approach (which I would support at a later point), we will need to update the default Dockerfile to include a shared volume

Reply:
About documenting this mount point:
do a note in hls outputs section of the admin manual can do the tricks, or do you think of an other place to mention it?
About docker:
I'm not familiar with it, but i'll check....

Reply:
We probably need to restructure https://libretime.org/docs/admin-manual/stream-configuration/ to have sub-pages for Icecast and HLS. Then each includes a walk through of how to set up and configure each. There are specific changes for each type that aren't generally relevant. For example changing the admin password in icecast or SSL support in HLS.

Reply:
i updated configuration.md but CI doesn't like it :(

Reply:
i just added the alias in the docker/nginx.conf, .... I don't know if it's enough

Reply:
@paddatrapper  sorry i've missed some important code
I've tested on vagrant it works on 1.4